### ISSUE TITLE
Adding a ButtonType=Literal button in HamburgerMenu breaks all other buttons

### ISSUE THREAD

**Issue by johanlindfors** [Windows-XAML/Template10#959]
I'm surprised that this happens...

If I add a button into Secondary Buttons, such as the following:

```
        <Controls:HamburgerButtonInfo ButtonType="Literal">
                <Grid Background="Red" Height="96"/>
        </Controls:HamburgerButtonInfo>
```

All other buttons will cease to function, they will trigger changes on the HamburgerMenu, but no Navigation will occur...

Bug or am I missing something?

Oh, and this is in the latest preview just pulled from Nuget, and also verified in the Hamburger template project on the master branch on GitHub.

**Comment by dg2k** [comment]
I've just found a bug @johanlindfors as an event is missing from Literal button DataTemplate
 See PR #962.

**Comment by johanlindfors** [comment]
I hope this fix makes it into the 1.1.11
